app/views/candidate_interface/application_form/show.html.erb

```diff
@@ -16,8 +16,8 @@
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
     <p class="govuk-hint govuk-!-margin-bottom-8 govuk-!-margin-top-2"><%= @application_form_presenter.updated_at %></p>
-    
-    <% if FeatureFlag.active?('apply_again') && @application_form.apply_2? && @previous_application_form.application_choices.rejected.present?  %>
+
+    <% if FeatureFlag.active?('apply_again') && @application_form.apply_2? && @previous_application_form&.application_choices&.rejected.present?  %>
```

Comment:
Using `&` is a bit of a smell, this might be more robust:

```suggestion
    <% if FeatureFlag.active?('apply_again') && @previous_application_form && @previous_application_form.application_choices.rejected.any?  %>
```

Reply:
Good suggestion. Thanks!